### ISSUE TITLE
UX improvement of website ~ opinions?

### ISSUE THREAD

**Issue by Neilblaze** [sugarlabs/www-sugarlabs#324]
![SG1](https://user-images.githubusercontent.com/48355572/86565156-f2e67800-bf84-11ea-9c4c-65873a9956ab.JPG)
![SG2](https://user-images.githubusercontent.com/48355572/86565162-f548d200-bf84-11ea-99fa-505a8e6d2a7a.JPG)
![SG3](https://user-images.githubusercontent.com/48355572/86565165-f679ff00-bf84-11ea-8473-3df9c2372709.JPG)
![SG4](https://user-images.githubusercontent.com/48355572/86565169-f843c280-bf84-11ea-89fa-bf2ebd132f42.JPG)
![SG5](https://user-images.githubusercontent.com/48355572/86565170-f8dc5900-bf84-11ea-9c2b-bad721306d25.JPG)

In the https://sugarlabs.org website, we can improve some of the elements mentioned below :-

- [ ] The floating social media icons are not perfectly aligned.

- [ ] On responsive view, the sugar-labs logo is not perfectly aligned with the hamburger nav-menu.

- [ ] On responsive view, the "searchForm" in the hamburger menu overlaps the sugar-labs logo.

- [ ] Improve the styling of nav-bar

- [ ] Remove "contribute to website" link at the footer.

- [ ] We can add custom "Back to top" CSS hover and remove the fixed "Back to top" from the footer.

- [ ] Implement smooth scrolling [optional]

Opinions?
I would love to work upon this :)
If agreed, can anyone assign me this?

**Comment by himanshujaidka** [comment]
Would you mind if I want to work with you?

**Comment by quozl** [comment]
This in conjunction with other recent issues and pull requests looks like a denial of service attack on our organisation.  Could you please explain why you have become interested in our web site?  Has someone asked you to be involved?  Is there a bounty?  Are we on some list somewhere?  I'd like to get us removed from any such list.

**Comment by Neilblaze** [comment]
@quozl No nothing such! Actually I was looking through the GCI'19 page where I came across Sugarlabs. Since the tech-stack of Sugarlabs mathches with mine, so I thought to look into the sugar labs website (https://www.sugarlabs.org). There while surfing the pages, I came across these experience issues which I believe that I can fix. That's why I created this issue #324 . Also, there are other issues which I am facing while running the sugarlabs website on my local, like styling and hover:effect issues.

And thus I want to work upon these :)

**And talking about your question, niether someone asked me to be involved nor there's any bounty.**
